Add Docker Build

yil532 opened the pull request.

yil532 committed:
add docker

bdwyer2 commented on .travis.yml:
```diff
@@ -1,7 +1,13 @@
+  - docker run -it -d -p 5000:5000 max-question-answering-web-app
   - pip install -r test-requirements.txt
+before_script:
+  - pip install -r requirements.txt
```
This should probably go in the `install` section

bdwyer2 replied:
Actually. I don't think this is needed at all. This will be installed inside the docker container, right?

yil532 committed:
delete requirements.txt

requirements.txt is used inside the docker container, docker will take care of installing the requirements inside itself.
